package.json

```diff
@@ -14,9 +14,10 @@
     "lint": "lerna run lint --stream",
     "prettier": "prettier --ignore-path .gitignore --write \"**/*.+(js|json)\"",
     "validate": "npm run prettier --list-different && npm run lint",
-    "version": "lerna version --conventional-commits --yes --no-commit-hooks --no-git-tag-version",
-    "publish": "lerna publish from-git --ignore-prepublish --no-commit-hooks --yes --no-verify-access --no-verify-registry",
-    "publish:ci": "lerna publish patch --yes",
+    "version": "lerna version --conventional-commits --yes --no-commit-hooks",
```

Comment:
Remove the `--no-git-tag-version`? Why? Are we  to rly on the package now?

Reply:
well I have just left this in there from @AlzAmido suggestions but don't think it will work for us in Monorepo of stacks. I'll change the version back to what it was originally but let's leave the pre:patch task in there for now

Reply:
If that's the case, the deploy needs to be from git too. Not from package.

Reply:
they are not being used right now the steps are ignored they are just there for potential usage and alignment with @AlzAmido  request.